tsl/test/expected/continuous_aggs_refresh.out

```diff
@@ -79,8 +79,13 @@ ORDER BY day DESC, device;
  Sat May 02 17:00:00 2020 PDT |      3 | 21.5217391304348
 (12 rows)
 
--- Refresh the rest
+-- Refresh the rest (and try DEBUG output)
+SET client_min_messages TO DEBUG1;
 CALL refresh_continuous_aggregate('daily_temp', '2020-05-01', '2020-05-03');
+DEBUG:  computed refresh window at [ 1588291200000000, 1588550400000000]
+DEBUG:  hypertable 1 existing  watermark >= new invalidation threshold 1588723200000000 1588550400000000
+DEBUG:  refreshing continuous aggregate "daily_temp" in window [ Thu Apr 30 17:00:00 2020 PDT, Sat May 02 17:00:00 2020 PDT ]
```

Comment:
Do you want to print debug messages? It looks like data that will change with each run.

Reply:
This was the reason I didn't previously. But you asked for this test case in order to know there aren't any crashes in the printing code :), so I added it. In this particular case, however,the output won't change every time, if I am not wrong. It is just printing information about the refresh window (which uses fixed timestamps).

Reply:
In that case, it should be fine. However, we probably need some way to send output like this to `/dev/null` (or the equivalent) so that we can check debug printouts for cases where the data *does* change.